app/src/main/res/layout/preference_cat_style.xml

```diff
@@ -7,7 +7,7 @@
     style="?android:attr/listViewStyle"
     android:layout_marginTop="16dp"
     android:textAlignment="viewStart"
-    android:layout_marginStart="16dp"
+    android:layout_marginStart="72dp"
```

Comment:
Since we're using 72dp in many screens, maybe it should be a generic dimen resource? Something like `@dimen/settings_keyline`

Reply:
I can open pr after this lands.